user-onboarding/cypress/e2e/form_test.cy.js

```diff
@@ -0,0 +1,273 @@
+const firstNameInput = () => cy.get('input[name="first_name"]')
+const lastNameInput = () => cy.get('input[name="last_name"]')
+const emailInput = () => cy.get('input[name="email"]')
+const passInput = () => cy.get('input[name="password"]')
+const confirmPassInput = () => cy.get('input[name="confirmPassword"]')
+const termsCheckbox = () => cy.get('input[name="terms"]')
+//const submitButton = () => cy.get('button[class="submit"]') - same as below:
+const submitButton = () => cy.get('.submit')
+const firstNameValErr = () => cy.get('div[class="error-first"]')
+const lastNameValErr = () => cy.get('div[class="error-last"]')
+const emailValErr = () => cy.get('div[class="error-email"]')
+const pwValErr = () => cy.get('div[class="error-pw"]')
+const confirmPwValErr = () => cy.get('div[class="error-confirmPw"]')
+
+describe('User Onboarding MVPs', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+
+  it('name input', () => {
```

Comment:
With testing name descriptions they should make sense. So here you could have
it('should allow user to input their name') or it('allows user to input name')